unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -441,15 +441,11 @@ public async UniTask<IReadOnlyList<FriendRequest>> GetFriendRequestsAsync(int se
         public void GetFriendsWithDirectMessages(string userNameOrId, int limit) =>
             apiBridge.GetFriendsWithDirectMessages(userNameOrId, limit, 0);
 
-        public FriendRequest GetAllocatedFriendRequest(string friendRequestId)
+        public bool GetAllocatedFriendRequest(string friendRequestId, out FriendRequest result)
```

Comment:
This pattern `bool` => `out value` usually has a"Try" prefix in the method name, I'd suggest renaming to `TryGetAllocatedFriendRequest`.